ee/agent/flags/flag_controller.go

```diff
@@ -767,3 +767,14 @@ func (fc *FlagController) UseCachedDataForScheduledQueries() bool {
 		WithDefaultBool(false),
 	).get(fc.getControlServerValue(keys.UseCachedDataForScheduledQueries))
 }
+
+func (fc *FlagController) SetCachedQueryResultsTTL(ttl time.Duration) error {
+	return fc.setControlServerValue(keys.CachedQueryResultsTTL, durationToBytes(ttl))
+}
+func (fc *FlagController) CachedQueryResultsTTL() time.Duration {
+	return NewDurationFlagValue(fc.slogger, keys.CachedQueryResultsTTL,
+		WithDefault(6*time.Hour),
+		WithMin(90*time.Minute),
```

Comment:
What is the downside to allowing the minimum to be lower?

Reply:
Ahh okay, l confused TTL with the interval with which we refresh the data ... this was my pre-coffee brain, your code is clear =)